### ISSUE TITLE
NumericEdit not working with @bind-Value

### ISSUE THREAD

**Issue by Andy74** [stsrki/Blazorise#360]
It seems impossible to edit a NumericEdit with decimal numbers. When trying to type 1.2 the value will be changed to 12. 
This behavior occurs, when binding the value with "bind-Value". When binding with "Value" then one can edit the number with ".".

Also at the start, the value will be shown with comma instead of point as separator.
So it seems, that here the formatting of the number will be done by the culture of the browser/system and not by the setting "DecimalsSeparator" (in my case I have german as my system culture).

here is my example code:
```cshtml
<NumericEdit TValue="double" @bind-Value="@dummyNumber1" DecimalsSeparator="."/>
@code 
{
    double dummyNumber1 { get; set; }
protected override async Task OnInitializedAsync()
    {
        dummyNumber1 = 42.5;
    }
}
```

**Comment by stsrki** [comment]
I just tried it and you're right it doesn't work. I will need to investigate more to see what's the actual problem. Thanks for reporting.

**Comment by stsrki** [comment]
I managed to make it work. The problem is actually bigger than expected and it's not so much in Blazorise as it's in the browsers support. Specifically in the localization of input fields. https://www.ctrl.blog/entry/html5-input-number-localization.html

To make it work I had to add support for `lang` attribute. So when defining the `NumericEdit` it also have to be defined. If left undefined it will fallback to `CultureInfo.InvariantCulture`. It acts more as a hint for Blazorise so it can know how to format the entered value.

**Example:**
```
<NumericEdit TValue="double" lang="en-us" @bind-Value="@dummyNumber1" DecimalsSeparator="." />
@code
{
    double dummyNumber1 { get; set; }
    protected override Task OnInitializedAsync()
    {
        dummyNumber1 = 42.5;

        return base.OnInitializedAsync();
    }
}
```

**Comment by Andy74** [comment]
today I updated Visual Studio, so that I now have netcore 3.1, which is necessary for blazories 0.8.7.1.
then I tried your example, but I still have the same issues with NumericEdit. (I use Chrome 78.0.3904.108). Has anybody any hints, what can be wrong?

**Comment by stsrki** [comment]
The issue is fixed in 0.8.8 which is still in development. It's not yet ready to be available. 

Also I will update Blazorise to 3.1 today.
